components/html-block/html-block.js

```diff
@@ -261,6 +261,7 @@ class HtmlBlock extends LitElement {
 	async _processEmbeds() {
 		const htmlFragment = document.createRange().createContextualFragment(this.html);
 		await renderEmbeds(htmlFragment);
+		this.updated(new Map([['embeds']]));
```

Comment:
I like the robustness here because it guarantees we'll always end up with the correct order of operations when rendering.